shared/p2p/connection_manager_test.go

```diff
@@ -0,0 +1,73 @@
+package p2p
+
+import (
+	"context"
+	"testing"
+
+	libp2p "github.com/libp2p/go-libp2p"
+	host "github.com/libp2p/go-libp2p-host"
+	inet "github.com/libp2p/go-libp2p-net"
+	peer "github.com/libp2p/go-libp2p-peer"
+	tu "github.com/libp2p/go-testutil"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+func hostWithConnMgr(t *testing.T) host.Host {
+	h, err := libp2p.New(context.Background(), optionConnectionManager(5))
+	if err != nil {
+		t.Fatal(err)
+	}
+	return h
+}
+
+// Test libp2p connection for connection manager
+type tconn struct {
+	inet.Conn
+
```

Comment:
remove line

Reply:
Hmm i dont know about that, the inet.Conn is a composition where the second group is individual fields

Reply:
ah , sorry I missed that. I mistook it for a separate var, its fine in that case